ext/calendar/easter.c

```diff
@@ -48,10 +53,25 @@ static void _cal_easter(INTERNAL_FUNCTION_PARAMETERS, bool gm)
 		}
 	}
 
-	if (gm && (year<1970 || year>2037)) {				/* out of range for timestamps */
+	#ifdef ZEND_ENABLE_ZVAL_LONG64
+	/* Compiling for 64bit, allow years between 1970 and 2.000.000.000 */	
+	if (gm && year<1970) {             /* timestamps only start after 1970 */
```

Comment:
Nit: while at it just fix the CS
```suggestion
	/* timestamps only start after 1970 */
	if (gm && year < 1970) {
```